src/lib/TranscriptEditor/MediaPlayer/index.js

```diff
@@ -159,15 +155,10 @@ class MediaPlayer extends React.Component {
     }
   }
 
-  /**
-   * @todo Consider refactoring using
-   * https://stackoverflow.com/questions/48048957/react-long-press-event
-   * To enable pressing on forward of backward key to move
-   */
   skipForward = () => {
     if (this.videoRef.current !== null) {
       const currentTime = this.videoRef.current.currentTime;
-      let newCurrentTime = currentTime + 5;
+      let newCurrentTime = currentTime + 10;
       newCurrentTime = Number((newCurrentTime).toFixed(1));
```

Comment:
this could be a new variable (const) too. best to avoid reassigning variables whenever possible

Reply:
ok, changing this